src/main/java/ro/societateahermes/backendservice/entities/DTO/MySubmissionDTO.java

```diff
@@ -0,0 +1,44 @@
+package ro.societateahermes.backendservice.entities.DTO;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import ro.societateahermes.backendservice.entities.form.Answer;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class MySubmissionDTO {
+
+    //fields related to the user
+    @NotBlank(message = "Email is mandatory")
+    @Pattern(regexp = "^\\w+([-+.']\\w+)*@\\w+([-.]\\w+)*\\.\\w+([-.]\\w+)*$", message = "Email invalid!")
+    private String email;
+    @NotBlank(message = "Username is mandatory")
+    private String username;
+    @NotBlank(message = "Firstname is mandatory")
+    @Pattern(regexp = "[a-zA-Z]+", message = "Firstname cannot contain numbers")
+    private String firstName;
+    @NotBlank(message = "Lastname is mandatory")
+    @Pattern(regexp = "[a-zA-Z]+", message = "Lastname cannot contain numbers")
+    private String lastName;
+    @NotBlank(message = "University is mandatory")
+    private String university;
+    @NotBlank(message = "Password is mandatory")
+    private String password;
+
+    //fields related to the submission of the form
+    private long formId;
+
+    private List<Answer> answers;
```

Comment:
AnswerDto

Reply:
Aici in dto -ul pentru submission imi poti explica de ce trebuie answerDto si nu answer?

Reply:
Pentru ca trimiti lista catre frontend si tot ce trimiti trebuie sa fie un dto. Acum e o lista cu entitatile din baza de date, ceea ce nu-i ok. In mapperul pentru Submission poti sa folosesti Mapperul pentru Answer ca sa convertesti lista de answeruri ale submisionului in AnswerDto